### ISSUE TITLE
spike viewer plot problem on Mac

### ISSUE THREAD

**Issue by ZhaoM22** [open-ephys/plugin-GUI#496]
Problem to Open files in OPEN Ephys GUI on the mac computer. 
The spike viewer showed just strange line without any spike waveform on my MacBook pro (see the below picture). The same file looks fine in the spike viewer on the Windows PC. I have download the newest Open Ephys GUI but coundn’t solve the problem. It is MacBook Pro (macOS Monterey v12.0.1).
Could you please let me know how to solve the problem? Thanks!
<img width="1644" alt="Mac_GUI_spikeview" src="https://user-images.githubusercontent.com/99220946/152881288-077fcb38-a406-4c52-aac3-56c471a38bdb.png">

**Comment by anjaldoshi** [comment]
Hi, 

Can you try to reproduce this error with v0.6.0 of the GUI? You can download v0.6.0 of the GUI from [here](https://openephysgui.jfrog.io/artifactory/Test/windows/open-ephys-v0.6.0-rc2-windows-beta.zip).  This version is still in 'beta', but we are doing the final release soon and I want to make sure if it got resolved in this version or not. 

\- Anjal

**Comment by ZhaoM22** [comment]
Hi Anjal,
The attached file is the Windows version (.exe). The problem happed on the Mac GUI. Could you please send me a linking of v0.60 of Mac GUI?
Thanks a lot!
Mingrui

From: Anjal Doshi ***@***.***>
Date: Thursday, February 10, 2022 at 4:47 PM
To: open-ephys/plugin-GUI ***@***.***>
Cc: Mingrui Zhao ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [open-ephys/plugin-GUI] spike viewer plot problem on Mac (Issue #496)

Hi,

Can you try to reproduce this error with v0.6.0 of the GUI? You can download v0.6.0 of the GUI from here<https://urldefense.proofpoint.com/v2/url?u=https-3A__openephysgui.jfrog.io_artifactory_Test_windows_open-2Dephys-2Dv0.6.0-2Drc2-2Dwindows-2Dbeta.zip&d=DwMCaQ&c=lb62iw4YL4RFalcE2hQUQealT9-RXrryqt9KZX2qu2s&r=QXyxb6vIHxBUE9chcVUw8pmHAQ7Z2-DoGMcGU8Ytt1k&m=WBJQYtuFiOYHrOLJkn3v5BJ-5oJXY9vAYeHD6Jp6S0PjiTbAbqIKKNNDclUovBGi&s=zM4icDB6jpN0sZ-CywQK7qAul3IuxudmHOKq01wGKGw&e=>. This version is still in 'beta', but we are doing the final release soon and I want to make sure if it got resolved in this version or not.

- Anjal

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_open-2Dephys_plugin-2DGUI_issues_496-23issuecomment-2D1035554960&d=DwMCaQ&c=lb62iw4YL4RFalcE2hQUQealT9-RXrryqt9KZX2qu2s&r=QXyxb6vIHxBUE9chcVUw8pmHAQ7Z2-DoGMcGU8Ytt1k&m=WBJQYtuFiOYHrOLJkn3v5BJ-5oJXY9vAYeHD6Jp6S0PjiTbAbqIKKNNDclUovBGi&s=COltTt49txgeidlORkzspfFTDfSrIvOK9R5rJ7iDlO0&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AXU73UUKXDQE36VCQENGMZLU2QW5LANCNFSM5OCEDNQQ&d=DwMCaQ&c=lb62iw4YL4RFalcE2hQUQealT9-RXrryqt9KZX2qu2s&r=QXyxb6vIHxBUE9chcVUw8pmHAQ7Z2-DoGMcGU8Ytt1k&m=WBJQYtuFiOYHrOLJkn3v5BJ-5oJXY9vAYeHD6Jp6S0PjiTbAbqIKKNNDclUovBGi&s=hEacM9nUirS61lsjHqokPZ9T6zI9ZbEBMmBaJfziDTY&e=>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMCaQ&c=lb62iw4YL4RFalcE2hQUQealT9-RXrryqt9KZX2qu2s&r=QXyxb6vIHxBUE9chcVUw8pmHAQ7Z2-DoGMcGU8Ytt1k&m=WBJQYtuFiOYHrOLJkn3v5BJ-5oJXY9vAYeHD6Jp6S0PjiTbAbqIKKNNDclUovBGi&s=6ufOGXREChsQgy6_4Vjk7PP6ufATQlacQRDtXu1pPN8&e=> or Android<https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMCaQ&c=lb62iw4YL4RFalcE2hQUQealT9-RXrryqt9KZX2qu2s&r=QXyxb6vIHxBUE9chcVUw8pmHAQ7Z2-DoGMcGU8Ytt1k&m=WBJQYtuFiOYHrOLJkn3v5BJ-5oJXY9vAYeHD6Jp6S0PjiTbAbqIKKNNDclUovBGi&s=MemNzpt-V1WXmf5OUv7F12iZMK8e6qldrUa893liWk8&e=>.

**Comment by anjaldoshi** [comment]
Sorry for the incorrect file. 
Here's the MacOS installer for v0.6.0: https://openephysgui.jfrog.io/artifactory/Test-Installer/mac/Open_Ephys_GUI_v0.6.0-rc2-beta.dmg
And, if you prefer the ZIP binaries, here it is: https://openephysgui.jfrog.io/artifactory/Test/mac/open-ephys-v0.6.0-rc2-mac-beta.zip

**Comment by jonessess** [comment]
Thanks- I was having the same problem with spike viewer in 0.5.5.3, so I tried the file you linked (beta of version 0.6.0), and it seemed to fix the display issue. 

However, the 0.6.0 pre-release version also crashed after only a few seconds of use, so I didn't test it thoroughly.

-Mark

**Comment by jsiegle** [comment]
What was the signal chain you were using when the crash occurred?

**Comment by jonessess** [comment]
I was using File Reader -> Bandpass Filter -> LFP Viewer -> Spike Detector -> Spike Viewer.

Upon further investigation, I found that the crashes seemed to be occurring soon after I changed the electrode list in Spike Detector under MacOS Big Sur 11.6.2. (I had deleted some stereotrode channels and added single electrode channels.)

On the other hand, when I tried the 0.6.0 pre-release with the same signal chain on a different machine using MacOS Catalina 10.15.7, I did not encounter any problems.

-Mark
